pkg/ui/workspaces/db-console/src/redux/nodes.ts

```diff
@@ -341,7 +342,11 @@ export function getDisplayName(
     return `${decommissionedString}(n${node.from.nodeID})`;
   }
   // as the only other type possible right now is INodeStatus we don't have a type guard for that
-  return `${decommissionedString}(n${node.desc.node_id}) ${node.desc.address.address_field}`;
+  if (includeAddress) {
```

Comment:
I'm assuming we don't have unit tests for this function - wondering if we want to add that? Up to you since we've done some end to end testing already on the drt cluster.

Reply:
done.